src/ios/CDVLocation.m

```diff
@@ -86,71 +86,82 @@ - (void)isLocationServicesEnabledWithCompletion:(void (^)(BOOL enabled))completi
 - (void)startLocation:(BOOL)enableHighAccuracy
 {
     __weak __typeof(self) weakSelf = self;
-    
+
     [self isLocationServicesEnabledWithCompletion:^(BOOL enabled) {
         __strong __typeof(weakSelf) strongSelf = weakSelf;
-        if (!strongSelf) {
-            return;
-        }
-        
+        if (!strongSelf) return;
```

Comment:
Do we still need the `strongSelf` reference?

Reply:
It mostly depends on if you wanna exit at this point or not. I'm fine with either approach

Reply:
That's not what I meant. Since we're no longer using `__locationStarted` or `__highAccuracyEnable` within the method, creating the `strongSelf` variable is unnecessary, as `weakSelf` will work just fine.

Reply:
Fixed.

Reply:
Where did `weakSelf` go?

Reply:
Fixed